docs/book/src/topics/clusterclass.md

```diff
@@ -0,0 +1,88 @@
+# ClusterClass
+
+- **Feature status:** GA
+- **Feature gate:** `MachinePool=true`, `ClusterTopology=true`
```

Comment:
MachinePool isn't required for all cluster class usage right? Just if you need machine pools?